components/Terminal.js

```diff
@@ -187,6 +172,47 @@ class Terminal extends Component {
     return this.showContent();
   };
 
+  /**
+   * autocomplete with the command the have the best match
+   * @param {object} input reference
+   */
+  autocompleteValue = (inputRef) => {
+    const { commands } = this.state;
```

Comment:
Use the descriptions instead and filter out any where the value is false so the ones we hide from help are also hidden from autocomplete